pulse_update/data/preprocess.py

```diff
@@ -0,0 +1,22 @@
+import os
+import csv
+from data import models
+
+def pull_data(output: str, connection_string: str) -> None:
+    os.makedirs(output, exist_ok=True)
+
+    domain_path = os.path.join(output, 'domains.csv')
+    subdomain_path = os.path.join(output, 'subdomains.csv')
+
+    with open(domain_path, 'w', newline='', encoding='utf-8') as domain_file, \
+         open(subdomain_path, 'w', newline='', encoding='utf-8') as subdomain_file:
+
+        domain_writer = csv.DictWriter(domain_file, fieldnames=['domain', 'filler', 'organization'])
```

Comment:
The "filler" heading is just a stand-in for now since the code that reads this resulting CSV does so by column index, not column name.